dashboard/config/scripts/Course4pre.script

```diff
@@ -71,8 +71,8 @@ level '4-5 Nested Loops 7'
 level '4-5 Nested Loops 11'
 level '4-5 Nested Loops 8'
 level '4-5 Nested Loops 12'
-assessment '4-5 Nested Loops Assessment 1'
-assessment '4-5 artist loops multi 1'
+level '4-5 Nested Loops assessment 1', assessment: true
```

Comment:
It looks like "Assessment" got downcased within some level names. Not 100% sure what this will break, but I think it is worth fixing.